tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
+  metadata:
+    name: cockpit-tasks
+  spec:
+    replicas: 1
+    selector:
+      infra: cockpit-tasks
+    template:
+      metadata:
+        labels:
+          infra: cockpit-tasks
+      spec:
+        containers:
+        - name: cockpit-tasks
+          image: docker.io/cockpit/tests
+          securityContext:
+            runAsUser: 1111
+            hostNetwork: false
+          env:
+          - name: TEST_JOBS
+            value: '8'
+          - name: TEST_PUBLISH
+            value: sink
+          volumeMounts:
+          - name: secrets
+            mountPath: "/secrets"
+            readOnly: true
+          - name: cache
+            mountPath: "/cache"
+            readOnly: false
+          - name: images
+            mountPath: "/cache/images"
+            readOnly: false
+          - name: shm
+            mountPath: "/dev/shm"
+            readonly: false
+          - name: cockpit-images
+          image: docker.io/cockpit/images
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+          - containerPort: 8443
+            protocol: TCP
+          volumeMounts:
+          - name: secrets
+            mountPath: "/secrets"
+            readOnly: true
+          - name: cache
+            mountPath: "/cache"
+            readOnly: false
+        volumes:
+        - name: secrets
+          secret:
+            secretName: cockpit-tests-secrets
+        - name: cache
+          hostPath:
+            path: "/var/cache/cockpit-tests"
+        - name: shm
+          emptyDir:
+            medium: Memory
+        - name: tmp
+          emptyDir: {}
+        serviceAccountName: cockpituous
+- kind: Service
```

Comment:
Like above, let's not do Images here, that's a separate service in the images/ directory.